dev/Deprecation.php

```diff
@@ -43,6 +43,16 @@ class Deprecation {
 	protected static $version;
 
 	/**
+	 * Override whether deprecation is enabled. If null, then fallback to 
+	 * SS_DEPRECATION_ENABLED, and then true if not defined.
+	 * 
+	 * Deprecation is only available on dev.
```

Comment:
Should probably add a note here that we can't use Config system because of infinite loop risk.